src/inputs/Select/index.tsx

```diff
@@ -64,7 +80,16 @@ function Select({ items, activeItemId, onItemClick, id, ...rest }: Props) {
             return (
               <MenuItem value={i.id} key={i.id}>
                 {i.iconUrl && <IconImg alt={i.label} src={i.iconUrl} />}
```

Comment:
what if it fails to fetch an icon?

Reply:
this is a case for many tokens in safe multisig, we need to be able to specify a default placeholder

Reply:
you can check my implementation for safe ethswap https://github.com/mikheevm/safe-app-tokenswap/blob/master/src/components/TokenIcon.tsx

Reply:
yes, makes sense, but what image should we use, this is a generic component, so I would guess that using a token image as a placeholder won't be useful in many cases. 

Perhaps we can add a new prop, `defaultPlacehoder`, and make it be a react component. 

wdty?

Reply:
it doesn't need to be a react component but an image

Reply:
but if they pass an URL, we can have the same problem. Or am I missing something?

Reply:
if they pass a react component which uses an invalid image url we can have the same problem

Reply:
there's no fully bulletproof solution

Reply:
ok, let's accept another URL to load in case the main URL fails.

Reply:
url or image source